source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,300 @@
+.. _minio-bucket-replication-resynchronize:
+
+
+========================================
+Resynchronize Bucket from Remote Replica
+========================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+The procedure on this page resynchronizes the contents of a MinIO
+bucket using a healthy replication remote. Resynchronization supports
+recovery after partial or total loss of data on a MinIO deployment in a 
+replica configuration.
+
+For example, consider a MinIO active-active replication configuration similar
+to the following:
+
+.. image:: /images/replication/active-active-twoway-replication.svg
+   :width: 600px
+   :alt: Active-Active Replication synchronizes data between two remote deployments.
+   :align: center
+
+Resynchronization allows using the healthy data on one of the participating
+MinIO deployments as the source for rebuilding the other deployment.
+
+Resynchronization is a per-bucket process. You must repeat resynchronization
+for each bucket on the remote which suffered partial or total data loss.
+
+.. admonition:: Support during BC/DR Operations
+   :class: important
+
+   `MinIO SUBNET <https://min.io/pricing?jmp=docs>`__ users can `log in <https://subnet.min.io/>`__ and create a new issue related to resynchronization. Coordination with MinIO Engineering via SUBNET can ensure successful resynchronization and restoration of normal operations, including performance testing and health diagnostics.
+
+   Community users can seek support on the `MinIO Community Slack <https://minio.slack.com>`__. Community Support is best-effort only and has no SLAs around responsiveness.
+
+.. _minio-bucket-replication-serverside-resynchronize-requirements:
+
+Requirements
+------------
+
+MinIO Deployments Must Be Online
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Resynchronization requires both the source and remote deployments be online and
+able to accept read and write operations. The source *must* have 
+complete network connectivity to the remote.
+
+The remote deployment may be "unhealthy" in that it has suffered partial or
+total data loss. Resynchronization addresses the data loss as long as both
+source and destination maintain connectivity.
+
+Disable Replication on Remote Bucket
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This procedure assumes the remote bucket has no existing replication
+configuration rules *or* that all rules are disabled. This ensures no unexpected
+or abnormal replication behavior during resynchronization.
+
+This procedure includes steps for configuring replication on the remote bucket
+once resynchronization completes.
+
+Replication Requires Matching Object Encryption
```

Comment:
@poornas @kannappanr attempting to address the SSE-KMS/S3 requirements here.

If this generally applies to all replication configs I'll use this in our other procedures as well.